<!-- -*- mode: XML -*- -->
<!--
 This work is sponsored by the Department of the Air Force under Air Force
 Contract FA8721-05-C-0002. Opinions, interpretations, conclusions, and
 recommendations are those of the author and are not necessarily endorsed by
 the United States Government.
-->

<launch>
  <arg name="throttle_axis" default="3" />
  <arg name="throttle_positive_gain" default="1" />
  <arg name="throttle_negative_gain" default="1" />
  <arg name="steering_axis" default="2" />
  <arg name="steering_positive_gain" default="-1" />
  <arg name="steering_negative_gain" default="-1" />
  <arg name="brake_axis" default="1" />
  <arg name="brake_positive_gain" default="1" />

  <node pkg="joy_to_servo" type="joy_to_tbs.py" name="joy_to_tbs">
    <param name="throttle_axis" value="$(arg throttle_axis)" />
    <param name="throttle_positive_gain" value="$(arg throttle_positive_gain)" />
    <param name="throttle_negative_gain" value="$(arg throttle_negative_gain)" />
    <param name="steering_axis" value="$(arg steering_axis)" />
    <param name="steering_positive_gain" value="$(arg steering_positive_gain)" />
    <param name="steering_negative_gain" value="$(arg steering_negative_gain)" />
    <param name="brake_axis" value="$(arg brake_axis)" />
    <param name="brake_positive_gain" value="$(arg brake_positive_gain)" />
  </node>
</launch>
